Update README.md for Project Euler (#3256)
TheAlgorithms/Python: project_euler/README.md

> I've tried to provide all the best possible solutions.

Reviewer: Seems weird. Maybe delete L8-9 or rephrase them?

> For description of the problem statements, kindly visit https://projecteuler.net/show=all

Reviewer: We don't need this, since we have L3?

## Solution Guidelines

> Welcome to [TheAlgorithms/Python](https://github.com/TheAlgorithms/Python)! Before reading the solution guidelines, make sure you read the whole [contributing guidelines](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md) as it won't be repeated in here. If you have any doubt on the guidelines, please feel free to [state it clearly in an issue](https://github.com/TheAlgorithms/Python/issues/new) or ask the community in [Gitter](https://gitter.im/TheAlgorithms). You can use the [template](https://github.com/TheAlgorithms/Python/blob/master/project_euler/README.md#solution-template) we have provided below as your starting point but be sure to read the [coding style](https://github.com/TheAlgorithms/Python/blob/master/project_euler/README.md#coding-style) part first.

Reviewer: ```suggestion
Welcome to [TheAlgorithms/Python](https://github.com/TheAlgorithms/Python)! Before reading the solution guidelines, make sure you read the whole [Contributing Guidelines](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md) as it won't be repeated in here. If you have any doubt on the guidelines, please feel free to [state it clearly in an issue](https://github.com/TheAlgorithms/Python/issues/new) or ask the community in [Gitter](https://gitter.im/TheAlgorithms). You can use the [template](https://github.com/TheAlgorithms/Python/blob/master/project_euler/README.md#solution-template) we have provided below as your starting point but be sure to read the [Coding Style](https://github.com/TheAlgorithms/Python/blob/master/project_euler/README.md#coding-style) part first.
```

### Coding Style

* Please maintain consistency in project directory and solution file names. Keep the following points in mind:

>   * Create a new directory only for the problems which does not exists yet.

Reviewer: ```suggestion
  * Create a new directory only for the problems which do not exist yet.
```

>   * If you create a new directory, please create an empty `__init__.py` file as well.

Reviewer: ```suggestion
  * If you create a new directory, please create an empty `__init__.py` file inside it as well.
```

>   * Please name the project directory as `problem_<problem_number>` where `problem_number` should be filled with 0s so as to occupy 3 decimal places. Example: `problem_001`, `problem_002`, `problem_067`, `problem_145`, and so on.

Reviewer: ```suggestion
  * Please name the project directory as `problem_<problem_number>` where `problem_number` should be filled with 0s so as to occupy 3 digits. Example: `problem_001`, `problem_002`, `problem_067`, `problem_145`, and so on.
```

* Please provide a link to the problem and other references, if used, in the module-level docstring.

* All imports should come ***after*** the module-level docstring.

* You can have as many helper functions as you want but there should be one main function called `solution` which should satisfy the conditions as stated below:
  * It should contain positional argument(s) whose default value is the question input. Example: Please take a look at [problem 1](https://projecteuler.net/problem=1) where the question is to *Find the sum of all the multiples of 3 or 5 below 1000.* In this case the main solution function will be `solution(limit: int = 1000)`.
  * When the `solution` function is called without any arguments like so: `solution()`, it should return the answer to the problem.

> * Every function, which includes all the helper functions if any and the main solution function, should have `doctest` in the function docstring along with a brief statement mentioning what the function is about.

Reviewer: ```suggestion
* Every function, which includes all the helper functions, if any, and the main solution function, should have `doctest` in the function docstring along with a brief statement mentioning what the function is about.
```

  * There should not be a `doctest` for testing the answer as that is done by our Travis CI build using this [script](https://github.com/TheAlgorithms/Python/blob/master/project_euler/validate_solutions.py). Keeping in mind the above example of [problem 1](https://projecteuler.net/problem=1):

  ```python
  def solution(limit: int = 1000):
      """
      A brief statement mentioning what the function is about.
      You can have a detailed explanation about the solution method in the
      module-level docstring.

      >>> solution(1)
      ...
      >>> solution(16)
      ...
      >>> solution(100)
      ...
      """
    ```

### Solution Template

> You can use the below template as your starting point but please read the [coding style](https://github.com/TheAlgorithms/Python/blob/master/project_euler/README.md#coding-style) first to understand how the template works.

Reviewer: ```suggestion
You can use the below template as your starting point but please read the [Coding Style](https://github.com/TheAlgorithms/Python/blob/master/project_euler/README.md#coding-style) first to understand how the template works.
```
